Dockerfile

```diff
@@ -2,20 +2,28 @@ FROM kbase/kb_jre:latest as build
 RUN apt-get -y update && apt-get -y install ant git openjdk-8-jdk make
 RUN cd / && git clone https://github.com/kbase/jars
 
-ADD . /src
-RUN cd /src && ant build
-RUN find /src
+COPY . /tmp/data_import_export
+
+ENV JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64/
```

Comment:
I am not sure why this env is needed, but if it's not there, the build complains that tools.jar isn't in JAVA_HOME, which is set to /usr/local/openjdk-8.  Maybe that's set in the parent image?  Anyway, setting it should be harmless.